source/frontend/src/features/properties/map/acquisition/add/models.ts

```diff
@@ -65,6 +73,12 @@ export class AcquisitionForm implements WithAcquisitionTeam {
     newForm.region = fromTypeCode(model.regionCode)?.toString();
     // ACQ file properties
     newForm.properties = model.fileProperties?.map(x => PropertyForm.fromApi(x)) || [];
+    newForm.fundingTypeCode = model.fundingTypeCode?.id;
+    newForm.fundingTypeOtherDescription = model.fundingOther || '';
+    newForm.project =
+      model.project !== undefined
```

Comment:
Hmm, if a model coming from the backend has a null or undefined id I'm not sure we want to set it to 0 here.

Bit of a weird case, this should never happen, but if it does happen and we just call context.Update(acqFile) on the backend we might start writing projects with id 0 to the db.

Reply:
That should not happen. In any case the toApi would clear it and be safe again

Reply:
> @FuriousLlama what is the plan for unit test coverage on this one? Do we need a follow up story?

Given that this story only modifies existing screens the current tests would work to get the snaps. Regarding the actual input of the forms. Do we want to test that using unittests? IMO that looks better suited for integration tests.

Reply:
Don't we have an entirely new api on the backend and frontend for the autocomplete though? Shouldn't we have some tests for that?

Reply:
Ah Backend tests, sure!